hack/dockerfile/binaries-commits

```diff
@@ -4,7 +4,7 @@ TOMLV_COMMIT=9baf8a8a9f2ed20a8e54160840c492f937eeaf9a
 
 # When updating RUNC_COMMIT, also update runc in vendor.conf accordingly
 RUNC_COMMIT=0351df1c5a66838d0c392b4ac4cf9450de844e2d
```

Comment:
Looks like runc was bumped in containerd to `74a17296470088de3805e138d3d87c62e613dfc4` - should we update accordingly?

Reply:
Bumped to the same version in #35439